lib/tasks/backup_db_rds.rake

```diff
@@ -6,18 +8,12 @@ task :backup_db_rds => :environment do
   current_time = Time.current.strftime("%Y%m%d%H%M%S")
 
   logger.info("Copying the database...")
-  backup_filename = "#{current_time}.rds.dump"
+  backup_filename = "#{Rails.env}-#{current_time}.rds.dump"
   system("PGPASSWORD='#{ENV["DIAPER_DB_PASSWORD"]}' pg_dump -Fc -v --host=#{ENV["DIAPER_DB_HOST"]} --username=#{ENV["DIAPER_DB_USERNAME"]} --dbname=#{ENV["DIAPER_DB_DATABASE"]} -f #{backup_filename}")
 
-  account_name = ENV["AZURE_STORAGE_ACCOUNT_NAME"]
-  account_key = ENV["AZURE_STORAGE_ACCESS_KEY"]
-
-  blob_client = Azure::Storage::Blob::BlobService.create(
-    storage_account_name: account_name,
-    storage_access_key: account_key
-  )
+  client = Aws::S3::Client.new(region: 'us-east-2')
```

Comment:
Is this the same as

```
client = ActiveStorage::Blob.services.fetch(:amazon)
```

Reply:
Basically. In this case I'm going straight to the client rather than doing it in a roundabout way through ActiveStorage.